doc-content/enterprise-only/project-data/delete-branches-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='delete-branches-proc']
+
+= Deleting branches
+You can delete any branch, except for the *master* branch because {CENTRAL} does not allow you to delete the *master* branch to avoid corrupting your environment. You must be in any branch other than *master* for the following procedure to work.
+
+.Procedure
+. Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
++
+.Deleting a branch menu
+image::getting-started/delete-branch-menu.png[Delete a branch menu]
+
+. Input the branch name to be deleted in the *Delete Branch* window and click *Delete Branch*.
```

Comment:
Really input and not select?

Reply:
Yup. You actually have to type it in exactly as it is named. No pull-down. I'm suggesting that dev fix this in the UI.